bin/rucio-cache-client

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
```

Comment:
Why remove it in just this file?

Reply:
mistake. phew another hour of tests running in travis :(